fvm/runtime/reusable_cadence_runtime.go

```diff
@@ -26,6 +39,39 @@ func NewReusableCadenceRuntime(rt runtime.Runtime, config runtime.Config) *Reusa
 		Environment: runtime.NewBaseInterpreterEnvironment(config),
 	}
 
+	blockRandomSource := stdlib.StandardLibraryValue{
```

Comment:
As mentioned above, we might want to consider moving this to Cadence

Reply:
Do you still think that would be better now that I added a comment describing why it is here.